app/services/usps_in_person_proofing/proofer.rb

```diff
@@ -46,7 +46,7 @@ def request_facilities(location, is_enhanced_ipp)
     # stored with the unique ID to be able to request the status of proofing.
     # @param applicant [Hash]
     # @return [Hash] API response
-    def request_enroll(applicant)
+    def request_enroll(applicant, is_enhanced_ipp)
```

Comment:
@gina-yamada (continuing the conversation here so it threads more neatly)
> [@eileen-nava](https://github.com/eileen-nava) Do you have a plan to test this in staging once merged in to confirm that the enrollment is EIPP? (ie How do you know this is an EIPP enrollment? At this stage- is it just a successful enrollment until we complete [LG-13577](https://cm-jira.usa.gov/browse/LG-13577)?

Unfortunately, I do think testing this in staging will have to be limited to confirming that the enrollment is successful.  Since we don’t yet save the sponsor_id in the in_person_enrollments table, (LG-13577) we can’t look at that to confirm that the enrollment was EIPP.  I also don’t know of any logs that we could use to confirm the enrollment was EIPP.

Let me know if you have any thoughts about how test in staging.

Reply:
This was my thinking. I wanted to confirm that was not a way I had not thought of. Sorry I forgot to comment in the code so the thread could be resolved.